src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -2,13 +2,14 @@
 // MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
 // ------------------------------------------------------------------------
 
+using System.Diagnostics.CodeAnalysis;
```

Comment:
Why did you move this using before other usings?

Reply:
My Cleanup Profile automatically does this.